lib/services/elasticsearch.js

```diff
@@ -1503,26 +1547,32 @@ class ElasticSearch extends Service {
       .then(({ body }) => {
         const successes = [];
 
-        for (let i = 0; i < body.items.length; i++) {
-          const item = body.items[i][Object.keys(body.items[i])[0]];
+        let
+          item,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "item" hides or potentially hides a variable declared in an outer scope at line 1365. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)